tools/cpboard.py

```diff
@@ -118,7 +118,7 @@ def reset(self):
         self.write(b'\r' + REPL.CHAR_CTRL_B) # enter or reset friendly repl
         data = self.read_until(b'>>> ')
 
-    def execute(self, code, timeout=10, async=False):
+    def execute(self, code, timeout=10, wait_for_response=False):
```

Comment:
Hmmm I merged this PR and then I went looking at it.  Isn't the sense of "wait_for_response" backwards from what it actually does?

Reply:
> Isn't the sense of "wait_for_response" backwards from what it actually does?

Right you are. 🤦 I've actually had a few of these edits in branches as my use of it progressed; I _think_ I had it proper at one point, but this definitely isn't. Fix imminent.